### PR TITLE
Mo hit hyperkit mount fix

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by dlorenc** [comment]
I merged in recent changes here, and updated godeps. Not sure the easiest way to go forward here.

**Comment by dlorenc** [comment]
Sorry, hit some godep issues. I have a big change here https://github.com/kubernetes/minikube/pull/2343 that would conflict, I'll get back to this one right after that goes in.

**Comment by mo-hit** [comment]
@dlorenc ahh okay, cool. Stoked for k8s 1.9 🤗

**Comment by dlorenc** [comment]
OK, I think I got this now.

**Comment by dlorenc** [comment]
@mo-hit ok, actually got it for real this time :)

**Comment by dlorenc** [comment]
@minikube-bot test this please

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -52,6 +52,8 @@ const (
 	humanReadableDiskSize = "disk-size"
 	vmDriver              = "vm-driver"
 	xhyveDiskDriver       = "xhyve-disk-driver"
+	NFSSharesRoot         = "nfs-shares-root"
```
Comment (r2d4) [inline cmd/minikube/cmd/start.go]: This needs some indicator that its macOS specific
@dlorenc

File: pkg/drivers/hyperkit/driver.go [hunk 2]
```diff
@@ -232,6 +250,56 @@ func (d *Driver) extractKernel(isoPath string) error {
...
+
+	if _, err := drivers.RunSSHCommandFromDriver(d, writeScriptCmd); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (d *Driver) nfsExportIdentifier(path string) string {
+	return fmt.Sprintf("docker-machine-driver-hyperkit %s-%s", d.MachineName, path)
```
Comment (dlorenc) [inline pkg/drivers/hyperkit/driver.go]: I should change this string to something minikube related instead of hyperkit.

**Pull request merged**